integration/src/test/resources/org/mule/test/integration/exceptions/on-error-nested-exceptions-config.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:test="http://www.mulesoft.org/schema/mule/test"
+      xsi:schemaLocation="
+       http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+       http://www.mulesoft.org/schema/mule/test http://www.mulesoft.org/schema/mule/test/current/mule-test.xsd">
+
+    <flow name="propagatesToOuterWithoutExceptionAndSucceeds">
+        <try>
```

Comment:
Double try is needed to test if an exception in an error handler is catched in an outer scope. Testing cases where it should return an exception or event was optional I suppose, just to make sure that in addition to handling it continue can turn the case into success.